src/calendar/actions/calendar-event/get-all-events-for-calendar.ts

```diff
@@ -0,0 +1,34 @@
+import { Fetcher } from '../../../common/api';
+import { endpoints } from '../../api/calendar-event/endpoints';
+import { ICalendarEvent } from '../../entities';
+import { ISearchResult } from '../../../common';
+import { ICalendarEventFilterDto } from '../../types';
+import { errorFactory, CalendarErr } from '../../errors/errors';
+import { StringUtils } from '../../../common/utils/string';
+
+export const getCalendarEventsByCalendarId = async (
+  fetcher: Fetcher,
+  calendarId: string,
+  filter: ICalendarEventFilterDto,
+): Promise<ISearchResult<ICalendarEvent>> => {
+  try {
+    const response = await fetcher.get<ISearchResult<ICalendarEvent>>(
+      StringUtils.bindContext(endpoints.GET_CALENDAR_EVENTS_WITH_CALENDAR_ID, {
+        calendarId,
+      }),
+      filter,
+    );
+
+    if (response.status >= 400) {
+      throw errorFactory.create(
+        CalendarErr.FETCH_CALENDAR_EVENT_WITH_CALENDAR_ID_FAILED,
```

Comment:
```suggestion
        CalendarErr.GET_CALENDAR_EVENT_WITH_CALENDAR_ID_FAILED,
```

Comme pour l'autre commentaire qui mentionne ce point, il serait intéressant de renommer l'erreur pour utiliser le même verbe que dans la méthode ("get").